src/test/java/com/cdancy/bitbucket/rest/features/RepositoryApiMockTest.java

```diff
@@ -88,7 +89,7 @@ public void testGetRepository() throws Exception {
             String repoKey = "myrepo";
             Repository repository = api.get(projectKey, repoKey);
             assertNotNull(repository);
-            assertTrue(repository.errors().size() == 0);
+            assertThat(repository.errors()).isEmpty();
```

Comment:
While things like this are fine they don't pertain to the PullRequest at hand. I'd say remove commit `5b792fb9bbcc30785bfd540931fd206244ce45aa` altogether.